### PR TITLE
Friendships

### PR DESCRIPTION
**Pull request by RadiactiveJesus**

### CODE REVIEW

**Review by imhta — CHANGES_REQUESTED** [review]
# Changes required 🛑 

### Hello  @RadiactiveJesus  , 👋 You have done a great job in this project 👏 
> It seems you are working alone, Did you inform it to student success team? If not kindly inform
* [ ] Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) 
* [ ] db migration, throws error fix that
image

----
### Don't forget to request another code review
----
**Pon Muthu Selvam N**
__nponmuthuselvam@gmail.com__
_If you have any queries, feel free to ping me on slack_

**Comment by RadiactiveJesus** [comment]
> # Changes required 🛑
> ### Hello @RadiactiveJesus , 👋 You have done a great job in this project 👏
> > It seems you are working alone, Did you inform it to student success team? If not kindly inform
> 
> * [ ]  Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
> * [ ]  db migration, throws error fix that
>   image
> 
> ### Don't forget to request another code review
> **Pon Muthu Selvam N**
> **[nponmuthuselvam@gmail.com](mailto:nponmuthuselvam@gmail.com)**
> _If you have any queries, feel free to ping me on slack_

Hello, yes Im working alone on the final Rails Project, student success is already informed about the situation. Thanks, on the other hand, ye  I willtake care of the description of the pull request

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required

Hello @RadiactiveJesus 

From the console, the friendship is OK. However, there are two errors that happens.
The first one was after accepting the friendship request, I click on `Home` link and I got this error
Screenshot_2020-02-06_15-04-14

Before that, I tried to access to an url and I got this error
Screenshot_2020-02-06_15-01-37

Also, please take a look on the comment below. The posts concern only the current user post's and his friend's post.

Kindly make the suggested changes and fixed the bug and then submit a new code review request.
Don't hesitate to write me on Slack if you have any problem.

Thanks

**Mael FOSSO**
[Github](https://github.com) | [Twitter](https://twitter.com) | [LinkedIn](https://www.linkedin.com/in/mael-elvis-fosso-650b6346/)

File: facebook/app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all
```
Comment (maelfosso) [inline facebook/app/controllers/home_controller.rb:8]: Here, things should change. You must no longer take all the posts but only created by the current user or by his friends

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great work implementing the requested changes.

**You may move on.**

**Good luck :four_leaf_clover: 
Simon Wathigo**

**Pull request merged**